server/channels/app/plugin_install_test.go

```diff
@@ -265,6 +266,69 @@ func TestInstallPluginLocally(t *testing.T) {
 			assertBundleInfoManifests(t, th, []*model.Manifest{manifest})
 		})
 	})
+
+	installPluginUpdateConfig := func(t *testing.T, th *TestHelper, id, version string, installationStrategy pluginInstallationStrategy) (*model.Manifest, map[string]any, *model.AppError) {
+		t.Helper()
+		var mp map[string]any
+
+		manifest := &model.Manifest{
+			Id:      id,
+			Version: version,
+		}
+		manifestJSON, jsonErr := json.Marshal(manifest)
+		require.NoError(t, jsonErr)
+		reader := makeInMemoryGzipTarFile(t, []testFile{
+			{"plugin.json", string(manifestJSON)},
+		})
+
+		actualManifest, appError := th.App.ch.installPluginLocally(reader, installationStrategy)
+		if actualManifest != nil {
```

Comment:
`actualManifest` should always be `!= nil`

Reply:
Clearly, I goofed on the logic here:

```
if actualManifest == nil {
    require.Equal(t, manifest, actualManifest)
}
```
my bad 😞